addons/mrp/models/stock_picking.py

```diff
@@ -53,6 +53,13 @@ def get_mrp_stock_picking_action_picking_type(self):
 class StockPicking(models.Model):
     _inherit = 'stock.picking'
 
+    has_kits = fields.Boolean(compute='_compute_has_kits')
+
+    @api.depends('move_lines')
+    def _compute_has_kits(self):
+        for picking in self:
+            picking.has_kits = any(b.type == "phantom" for b in picking.move_lines.bom_line_id.bom_id)
```

Comment:
on move_lines `description_bom_line` is always computed in the view. For perf, it is better to check with this field.
```python
@api.depends('move_lines')
def _compute_has_kits(self):
    for picking in self:
        picking.has_kits = bool(picking.move_lines.mapped('description_bom_line'))
```